demo/src/lib.rs

```diff
@@ -241,6 +245,11 @@ impl RenderOptions {
             combo.end(ui);
             self.tonemapper_type = current_tonemapper_type.into();
         }
+
+        ui.checkbox(
+            imgui::im_str!("freeze visibility"),
+            &mut self.freeze_visibility,
```

Comment:
Would probably rename this to `enable_visibility_update` and default it to `true`
That would keep consistency with other settings in this menu (checked is active, unchecked is off) + it means that you wouldn't need to negate it when you copy it into `VisibilityConfig`.